_presenters/mike-hoolehan.md

```diff
@@ -5,11 +5,11 @@ layout: speaker-template
 mastodon: null
 name: Mike Hoolehan
 permalink: /presenters/mike-hoolehan/
-photo_url: ''
+photo_url: /static/img/presenters/mike-hoolehan.jpg
 pronouns: he/him
 slug: mike-hoolehan
 twitter: null
 website: null
 ---
 
-Mike is a software developer living in Tallinn, Estonia.  Although his list of former hobbies is expansive and every-growing, his love of of software is constant keeps him curious and enthusiastic after 20 years of professional software development across numerous languages and frameworks. Currently, Mike is following his bliss with Django, Flutter, bike-touring, camping, and tabletop gaming.
+Mike is a software developer living in Tallinn, Estonia.  Although his list of former hobbies is expansive and every-growing, his love of software is constant keeps him curious and enthusiastic after 20 years of professional software development across numerous languages and frameworks. Currently, Mike is following his bliss with Django, Flutter, bike-touring, camping, and tabletop gaming.
```

Comment:
I think there might be another word missing: "is constant and keeps" ?

Reply:
@KatherineMichel  thanks so much!